src/open_dread_rando/dread_patcher.py

```diff
@@ -6,30 +6,32 @@
 from construct import ListContainer
 from mercury_engine_data_structures.file_tree_editor import OutputFormat
 
-from open_dread_rando import elevator, game_patches, lua_util
 from open_dread_rando.constants import FadeTimes
 from open_dread_rando.cosmetic_patches import apply_cosmetic_patches
```

Comment:
This need to be `from open_dread_rando.cosmetic_patches.cosmetic_patches ...` now?

Reply:
oh another review...so this is a duplicate now :)